### DISCUSSION TITLE
New charts release pipeline

### DISCUSSION THREAD

**Discussion by Aisuko** [meshery/meshery#1792] [Ideas] [answered]
As we saw that we support as more as the Service Mesh adapters, so we need to add the helm charts release pipeline. I saw that the Helm community released two tools related to this topic. And I will post the information about the tools to here.

**Comment by kushthedude** [comment]
We can use https://github.com/helm/chart-releaser-action to turn our gh-pages branch to chart repository and this can be infused in our CI/CD pipelines too.
/cc @leecalcote

**Comment by leecalcote** [comment, marked as answer]
Meshery is currently using stefanprodan/helm-gh-pages wrapped by some additional logic in https://github.com/layer5io/meshery/blob/master/.github/workflows/helm-chart-releaser.yml
